src/components/sourcepublisher/SourcePublisher.js

```diff
@@ -49,14 +49,15 @@ function publishLayer(data, type, name, url) {
   if (!schema) {
     throw Error(`No schema for ${type}`);
   }
+  // Should we just compile everything so we can cross reference?
+  // We need DATASET for requestInit in rasters.
```

Comment:
We do compile all schemas, I think. What is being asked here?

Reply:
Vestigial comment.